teaspoon-devkit.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency "aruba"
   s.add_dependency "appraisal"
   s.add_dependency "simplecov"
+  s.add_dependency "selenium-webdriver"
```

Comment:
This still isn't a requirement -- but we could maybe update the readme? phantomjs is still an option for using as a driver, and it might be annoying to require `selenium-webdriver` if that's what you're using. Open to opinions.

Reply:
yes, I'm just testing, but I think the issue is only on the CI, if I got anything, I'll let you know

Reply:
ah, ok. haha. try taking out the line it's failing on -- maybe we don't need to do that anymore?

Reply:
now is green, but I will check about the broken test, let me know what do you think